src/Sulu/Bundle/AdminBundle/Resources/js/containers/DeleteDependantsDialog/DeleteDependantsDialog.js

```diff
@@ -0,0 +1,223 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {action, computed, observable} from 'mobx';
+import Dialog from '../../components/Dialog';
+import ProgressBar from '../../components/ProgressBar';
+import ResourceRequester from '../../services/ResourceRequester';
+import RequestPromise from '../../services/Requester/RequestPromise';
+import {translate} from '../../utils';
+import styles from './deleteDependantsDialogStyles.scss';
+import type {Resource} from '../../types';
+
+type Props = {
+    dependantResources: Resource[][],
+    dependantResourcesCount: number,
+    onCancel?: () => void,
+    onClose?: () => void,
```

Comment:
think its a bit confusing that there is a `onCancel` and `onClose` prop. would just add `onCancel` to match the props of the default `Dialog`

Reply:
Closing and cancelling are different actions here. You can only close, if the deletion has already succeeded, if it's in progress, it's a cancel